# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# Filename: package/.../lvm2/dietlibc.patch
# Copyright (C) 2007 The OpenSDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

--- ./lib/misc/lib.h.orig	2007-07-18 21:00:06.000000000 -0400
+++ ./lib/misc/lib.h	2007-07-18 21:00:36.000000000 -0400
@@ -31,5 +31,6 @@
 #include "lvm-wrappers.h"
 
 #include <libdevmapper.h>
+#include <stdint.h>
 
 #endif
